packages/core/src/codewhisperer/util/authUtil.ts

```diff
@@ -345,6 +348,30 @@ export class AuthUtil {
         }
     }
 
+    public async notifySessionConfiguration() {
+        const suppressId = 'amazonQSessionConfigurationMessage'
+        const settings = AmazonQPromptSettings.instance
+        const shouldShow = await settings.isPromptEnabled(suppressId)
+        if (!shouldShow) {
+            return
+        }
+
+        const message = localize(
```

Comment:
This is probably a stupid question but will this show up to users who already have extended sessions on their IdC enabled?

Reply:
yes it will, so we will only show it once as a FYI

Reply:
Can we detect if an IdC already has extended session enabled?

I'm guessing the team that owns the IdC might not be the team necessarily using this extension so having a dev (or multiple devs) message the IdC team to ask if extended sessions it enabled could annoy them

Reply:
the session duration is controlled by the Q admin, product's idea is that users can contact their admin to configure the extended session if this is something that they want as end users

Reply:
Right, I was more concerned with notifying customers who already have extended session duration enabled. AFAICT we will also be notifying them and saying "hey tell your admin to enable extended session duration" even though its already enabled for them

Reply:
agreed, but I don't think we need to get into checking end user's session configuration just for the notification purpose, it's something that they can easily check with their admin. This is what we aligned with product.